src/init.d/002-run-as-prerequisites.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+if [ -n "${RUN_AS}" ] ; then
+    if ! command -v sudo &> /dev/null; then
+        echodt "ERROR: sudo package is not installed, please install it 'apk add --no-cache sudo'"
```

Comment:
Whalesome is not alpine specific. We might instruct the user to use the "pkg" skript. See https://github.com/sha512man/whalesome/blob/master/src/bin/pkg